MUXSDKKaltura/MUXSDKKalturaTests/MUXSDKStatsTests.swift

```diff
@@ -172,4 +172,133 @@ class MUXSDKStatsTests: XCTestCase {
         )
         self.assertDispatchedCustomerViewerDataEventsAtIndex(index: 0, expectedCustomerViewerData: expectedViewerData)
     }
+    
+    func testDestroyPlayer() {
+        guard
+            let customerData = MockedData.customerData
+        else {
+            XCTFail("Customer data not found")
+            return
+        }
+       
+        // Start monitor player to emit events
+        MUXSDKStats.monitorPlayer(player: MockedData.player, playerName: MockedData.playerName, customerData: customerData)
+       
+        // Remove observers on the player
+        MUXSDKStats.destroyPlayer(name: MockedData.playerName)
+
+        let expectedEvents = [
+            MUXSDKPlaybackEventViewInitEventType,
+            MUXSDKDataEventType,
+            MUXSDKPlaybackEventPlayerReadyEventType,
+            MUXSDKPlaybackEventViewEndEventType
+        ]
+
+        // Assert expected events
+        self.assertDispatchedEventTypesMatch(
+            expectedEventTypes: expectedEvents,
+            for: MockedData.playerName
+        )
+
+        // Clean player instance after test
+        MUXSDKStats.destroyPlayer(name: MockedData.playerName)
```

Comment:
Why do you need this twice

Reply:
First one is for detaching the player for the test and include all the expected events and the second is for cleaning the player instance after the test.